tls/s2n_config.c

```diff
@@ -404,3 +405,11 @@ int s2n_config_set_client_hello_cb(struct s2n_config *config, s2n_client_hello_f
 
     return 0;
 }
+
+int s2n_config_set_max_fragment_length(struct s2n_config *config, uint8_t mfl_code)
+{
+    config->mfl_code = mfl_code;
```

Comment:
Good call.
Should we alert (print) the user but not terminate the handshake in case of invalid input as we discussed above?

```
if (mfl_code >= S2N_TLS_MAX_FRAG_LEN_INVALID) {
       fprintf(stderr, "invalid Max Fragmentation Length encountered"\n");
    }
```